### PR TITLE
Update for Linter fix

### PR DESCRIPTION
**Pull request by cancelei**
Hello Reviewer, 

In this Capstone project, I created a Website with 2 HTML files, 1 CSS, and 1 Javascript file. They work all together.

The project is about a Permaculture Event that will happen later this year in Harvard Square, it has featured speakers that are the highlight of this event.

**Project Highlights**

- JS object generated Speakers Section.
- Multiple menus and adapted to mobile and Desktop Screens.
- Linters Passing without errors.

Happy review!

### CODE REVIEW

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: DEPLOYMENT REQUIREMENTS
- [x]  You are expected to make use of GitHub Pages to deploy the web pages, please kindly do this :+1: 
- [x] The project is expected to be deployed properly, all pages, links, and images working as expected.
- [x] Add a link to the deployed project in your README preferably where you have `live demo`.

File: README.md [hunk 1]
```diff
@@ -138,7 +153,7 @@ N/A
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-Ed and Jav helped me in concluding this project, they are part of my current working and standup teams.
+N/A
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
```
Comment (Efiamotu-1) [inline README.md]: - This is an optional section and as such if no `faq` is written you are required to remove this section :+1: 
- please kindly ensure you fill in details for the `Contributing & Show your support` section :+1:

File: README.md [hunk 2]
```diff
@@ -138,7 +153,7 @@ N/A
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (Efiamotu-1) [inline README.md:154]: - It is important that your "Acknowledgements" section gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [ ] please fix the broken link to your license file as it is currently showing a `404` error when clicked :+1:

File: dist/index.html [hunk 3]
```diff
@@ -91,18 +106,18 @@ <h4 class="cardtitle">Motivation on Farm's</h4>
   <!-- Footer Section with Partner Information -->
   <footer class="sectionfooter">
```
Comment (Efiamotu-1) [inline dist/index.html]: - [ ] From your current design as shown below you do not have the appropriate footer for the home page, what you have specified as the footer should be the partners, the footer should match the template requirement as shown below.
- please it is important to closely design your project to match the provided design as this shows how much attention to details you pay as a developer and it is an awesome quality to have :+1: 
- ## figma design
i-f
- your design
i-fs

Comment (Efiamotu-1) [inline dist/index.html]: - [ ] From your current design as shown below you do not have the appropriate footer for the home page, what you have specified as the footer should be the partners, the footer should match the template requirement as shown below.
- please it is important to closely design your project to match the provided design as this shows how much attention to details you pay as a developer and it is an awesome quality to have :+1: 
- ## figma design
i-f
- your design
i-fs

File: dist/about.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    <br>
+    <div class="eventphotoparent">
+      <img src="/Assets/About/about_2011.png" alt="Event in 2011">
+      <img src="/Assets/About/about_2013.png" alt="Event in 2013">
+    </div>
+<br>
+<button class="btnspeaker">MORE</button>
+<br><br>
+  </section>
+  <footer class="sectionfooter">
```
Comment (Efiamotu-1) [inline dist/about.html]: - [ ] From your current design as shown below you do not have the appropriate footer for the about page, what you have specified as the footer should be the partners, the footer should match the template requirement as shown below. :+1: 
- ## figma design
a-f

File: dist/about.html [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    <h3 class="partner mainTitle">Partner</h3><br>
```
Comment (Efiamotu-1) [inline dist/about.html]: - This image below is the footer design for the mobile version as provided in the template design from cindy shin which you can access with this [link](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). please ensure to add this to your project before requesting for a new review :+1: 
- figma design
ma-f

File: dist/about.html
Comment (Efiamotu-1) [inline dist/about.html:1]: - [ ] The about page of your design doesn't completely match the provided template design, i have used the red lines as shown in the images to show you areas where they are inconsistent in spacing so you can fix them appropriately so your design matches the provided template :+1: :sparkles: 
- template design
spp
- your design
pp

File: dist/index.html
Comment (Efiamotu-1) [inline dist/index.html:1]: - [ ] The home page of your design doesn't completely match the provided template design, i have used the red lines as shown in the images to show you areas where they are inconsistent in spacing so you can fix them appropriately so your design matches the provided template :+1: :sparkles: 
- template design
home-d
- your design
home-s

Comment (Efiamotu-1) [inline dist/index.html:1]: - [ ] please i kindly suggest you fix the design of the home page for mobile screen becasue as shown from the images below it doesn't match.
- template design
mob-home
- your design
mob-home-s

Comment (Efiamotu-1) [inline dist/index.html:1]: - [x] please notice the spacing between each nav link as shown by the red lines, kindly make these corrections to your project :+1: 
- template design
sp-f
- your design
sp

File: dist/about.html
Comment (Efiamotu-1) [inline dist/about.html:1]: - [x] please your design for the about page on mobile is not consistent with the provided design template kindly work on your design before resubmitting :+1: 
- template design
abb
- your design
abb-

Comment (Efiamotu-1) [inline dist/about.html:1]: - [x] This section is supposed to display two pictures on an horizontal line side by side each other not one on top of the other on the desktop version howver for your design you have it vertically aligned one above the other on desktop version, please kindly fix this 
- template design
ss-f

Comment (Efiamotu-1) [inline dist/about.html:1]: - The template design as shown beloiw does not have a partners section for the desktop view of the about me page but as shown below your design has a partners sectionn on desktop view, please kindly fix this :+1:

File: dist/index.html
Comment (Efiamotu-1) [inline dist/index.html:1]: - [x] Your speaker section on the mobile version has text overlapping please kindly fix this as it is affecting the design of your project on mobile versions :+1: 
- your design
sep
- [x] As shown in the image below your home page mobile version should not have a partners section please kindly fix this. 
template design
p

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @cancelei , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @cancelei,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Professional readme file ✔️
- No linter error ✔️
- Amazing design ✔️
209362895-9762f2f8-13cf-4ce0-90b6-a672c4f04196

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**